sale_timesheet_draft_invoice/models/account_invoice_line.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+from odoo.osv import expression
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice.line'
+
+    @api.model
+    def create(self, vals):
+        """
+        Override to link to 'timesheet_invoice_id' the invoice
+        """
+        invoice_line = super().create(vals)
+        if (
+            invoice_line.invoice_id.type == 'out_invoice'
+            and invoice_line.invoice_id.state == 'draft'
+        ):
+            invoice_line.link_timesheets_lines()
+            return invoice_line
```

Comment:
:fire: ORM broken -> you have to return the line anyway

Reply:
our integral  tests was broken due to this mistake (see traceback under),
start to fails on loading items from yaml file, but test in OCA pass successfully, that means yaml excluded from loading in OCA build?

     Traceback (most recent call last):
     File "/odoo/src/odoo/modules/registry.py", line 85, in new
     odoo.modules.load_modules(registry._db, force_demo, status, update_module)
     File "/odoo/src/odoo/modules/loading.py", line 380, in load_modules
     loaded_modules, update_module, models_to_check)
     File "/odoo/src/odoo/modules/loading.py", line 274, in load_marked_modules
     perform_checks=perform_checks, models_to_check=models_to_check
     File "/odoo/src/odoo/modules/loading.py", line 184, in load_module_graph
     _load_data(cr, module_name, idref, mode, kind='demo')
     File "/odoo/src/odoo/modules/loading.py", line 95, in _load_data
     tools.convert_file(cr, module_name, filename, idref, mode, noupdate, kind, report)
     File "/odoo/src/odoo/tools/convert.py", line 789, in convert_file 
     convert_yaml_import(cr, module, fp, kind, idref, mode, noupdate, report)
     File "/odoo/src/odoo/tools/yaml_import.py", line 854, in yaml_import 
     yaml_interpreter.process(yaml_string)
     File "/odoo/src/odoo/tools/yaml_import.py", line 797, in process
     self._process_node(node)
     File "/odoo/src/odoo/tools/yaml_import.py", line 810, in _process_node
     self.process_python(node)
     File "/odoo/src/odoo/tools/yaml_import.py", line 601, in process_python
     unsafe_eval(code_obj, {'ref': self.get_id}, code_context)
     File "/odoo/src/addons/account/demo/account_invoice_demo.yml", line 111, in <module>
     self._update('account.invoice.line', 'account', line_vals, 'ceo_wages_line')
     File "/odoo/src/odoo/addons/base/ir/ir_model.py", line 1502, in _update
     for parent_model_name, parent_field in current_model._inherits.items():
     AttributeError: 'NoneType' object has no attribute '_inherits'

cc @pedrobaeza